pkg/blueprint/disk_customizations.go

```diff
@@ -64,6 +66,15 @@ type PartitionCustomization struct {
 	// (optional, defaults depend on payload and mountpoints).
 	MinSize uint64 `json:"minsize" toml:"minsize"`
 
+	// The partition type GUID for GPT partitions. For DOS partitions, this
+	// field can be used to set the partition type (e.g. "swap").
+	// If not set, the code will make the best guess based on the mountpoint or
+	// the payload type.
+	// Examples:
+	// 3B8F8425-20E0-4F3B-907F-1A25A76F98E8 (/srv on GPT)
+	// 06 (FAT16 on DOS)
+	GUID string `json:"guid,omitempty" toml:"guid,omitempty"`
```

Comment:
I don't like calling this a GUID when it isn't always a GUID. In other places this is called Type, but that's obviously already been used. Maybe `PartType`?

Reply:
This got renamed to PartType in https://github.com/osbuild/images/pull/1115/files and is no longer needed in this PR.